Finmer.Core/Serialization/IFurballContentReader.cs

```diff
@@ -21,6 +21,11 @@ namespace Finmer.Core.Serialization
     public interface IFurballContentReader
     {
 
+        /// <summary>
+        /// Returns the format version of the module file being read.
+        /// </summary>
+        int GetVersion();
```

Comment:
Probably pedantic, but I would prefer this function to be called `GetFormatVersion`, so that it is explicit about _what_ version it's returning - the method name as-is is a bit generic.